File: Pattern 6/Pattern_6.java
class Solution {

    void printTriangle(int n) {
        // simple pattern just iterate on the loop 
	    // print on every row with values is 1 to i
	    
	    for(int i =n; i>0; i--) {
	        for(int j =1; j<=i; j++) {
	            System.out.print(j+" ");
	        }
	        System.out.println();
	    }
    }
}

File: Pattern 6/Pattern_6.py
class Solution:
    def printTriangle(self, n):
        # simple pattern just iterate on the loop 
	    # print on every row with values is 1 to i
	    
	    for i in range(n,0,-1):
	        for j in range(1,i+1):
	            print(j,end= " ")
	        print()
	    